### DISCUSSION TITLE
How to create floating notification / Head-up with react native expo

### DISCUSSION THREAD

**Discussion by rickyanwar** [expo/expo#18171] [Archived: Questions]
![image](https://user-images.githubusercontent.com/36995966/178108445-456abd2f-0501-4d85-8b73-c58d20485423.png)
i was created local notification without other third party, My expectation is that notifications are like whatsapp message floating and there are actions button,success show notification but not floating/popup notification

i was created local notification without other third party, My expectation is that notifications are like whatsapp message floating and there are actions button,success show notification but not floating/popup notification

```

Notifications.setNotificationChannelAsync('location-reached', {
    name: 'location-reached',
    importance: Notifications.AndroidImportance.MAX,
    vibrationPattern: [0, 250, 250, 250],
    lightColor: '#FF231F7C',
});

Notifications.setNotificationHandler({
    handleNotification: async () => ({
        shouldShowAlert: true,
        shouldPlaySound: true,
        shouldSetBadge: true,
        priority: "high",

    }),
});

export const sendNotification = async (address) => {

    const id = await Notifications.scheduleNotificationAsync( {
        content: {
            title: `Destination will arrive soon`,
            body: address,
            sound: 'default',
            vibrate: true,
        },
        trigger: {
            seconds: 1,
            channelId: 'location-reached'
        },
    });
    console.log("notif id on scheduling", id)
    return id;
}
```
i was give permission for this notif, but not showing floating notif
![image](https://user-images.githubusercontent.com/36995966/178108480-4680325c-f75a-4154-8e82-c95ab2e9d1ef.png)

**Comment by b0iq** [comment]
You could use this: 
https://github.com/calintamas/react-native-toast-message

**Reply by rickyanwar** [reply]
I think it notification not toast

**Reply by b0iq** [reply]
Yes, You should've listener for in-app notification and present it the way you like, Toast, Alert...etc

**Reply by rickyanwar** [reply]
I think toast can't present in background
